app/assets/stylesheets/modules/forms.scss

```diff
@@ -3,8 +3,19 @@
     padding-top: 7px;
   }
 
+  hr {
+    margin-top: 20px;
+    margin-bottom: 20px;
+    border: 0;
+    border-top: 1px solid #d5d5d3;
+  }
+
   .or-divider {
     text-align: center;
+    width: 3em;
+    margin: -32px auto 10px auto;
+    background-color: #fbf9f9;
```

Comment:
Color literals like `#fbf9f9` should only be used in variable declarations; they should be referred to via variable everywhere else.